homeassistant/components/hangouts/hangouts_bot.py

```diff
@@ -78,49 +81,73 @@ def async_update_conversation_commands(self, _):
         self._conversation_list.on_event.add_observer(
             self._handle_conversation_event)
 
+    def async_handle_update_error_suppressed_conversations(self, _):
+        """Resolve the list of error suppressed conversations."""
+        self._error_suppressed_conv_ids = []
+        for conversation in self._error_suppressed_convs:
+            conv_id = self._resolve_conversation_id(conversation)
+            if conv_id is not None:
+                self._error_suppressed_conv_ids.append(conv_id)
+
     def _handle_conversation_event(self, event):
```

Comment:
Is this called from an async context? If so, let's mark it as a `callback` and add `async` in the name.